setup.ts

```diff
@@ -45,6 +47,28 @@ async function accessAndRefreshTokens(
   }
 }
 
+function transactionLogPath(logPath: string | null): void {
+  const config = dotenv.parse(fs.readFileSync(".env"));
+  config.TRANSACTION_LOG_PATH =
+    logPath ?? path.resolve(__dirname, "logs/transaction.log");
+  fs.writeFileSync(".env", "");
+  for (const key in config) {
+    fs.appendFileSync(".env", `${key}=${config[key]}\n`);
```

Comment:
Verify that the file can be written to by the setup user.

Reply:
Done.